cime_config/machines/cmake_macros/gnu_cades.cmake

```diff
@@ -1,3 +1,3 @@
-string(APPEND FFLAGS " -O -fno-range-check")
+string(APPEND CMAKE_Fortran_FLAGS " -O -fno-range-check")
```

Comment:
I understand this PR is just translating old syntax into new syntax, but...what's the point of `-O`?

Reply:
No idea. I don't know if the machine `cades` is even something we still use.

Reply:
We could email the person from the SUPPORTED_BY xml entry and ask. Not urgent, though.